hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/NodeEndpoint.java

```diff
@@ -108,6 +115,10 @@ public Response getDatanodes() {
           if (datanode.getUuid().equals(pipeline.getLeaderId())) {
             leaderCount.getAndIncrement();
           }
+          int openContainerPerPipeline =
+                  reconContainerManager.getPipelineToOpenContainer()
+                  .getOrDefault(pipelineID, 0);
+          openContainers.set(openContainers.get() + openContainerPerPipeline);
```

Comment:
Doesn't seem atomic to me.. Try this?
```suggestion
          openContainers.getAndAdd(openContainerPerPipeline);
```